lib/auth/init.go

```diff
@@ -174,15 +173,15 @@ func Init(cfg InitConfig) (*AuthServer, *Identity, error) {
 			return nil, nil, trace.Wrap(err)
 		}
 	}
-	if firstStart {
-		if len(cfg.ReverseTunnels) != 0 {
-			log.Infof("FIRST START: Initializing reverse tunnels")
-			for _, tunnel := range cfg.ReverseTunnels {
-				if err := asrv.UpsertReverseTunnel(tunnel, 0); err != nil {
-					return nil, nil, trace.Wrap(err)
-				}
+	if len(cfg.ReverseTunnels) != 0 {
```

Comment:
so if there's no `firstStart` logic here it means there's no way for me to preconfigure tunnels and then later remove them.